reference/content-types/smart_content.rst

```diff
@@ -36,9 +39,114 @@ Parameters
         choose between the elements in this collection. The selected value is
         also passed to the HTML template.
 
+.. note::
+
+    "properties" can include structure properties or extension Data:
+
+    * title - is a property of the template
+    * excerpt.title - is a property of the excerpt template extension with the name title
+
+Return Value
+------------
+
+.. list-table::
+    :header-rows: 1
+
+    * -
+      - Name
+      - Type
+      - Description
+    * - content
+      - uuid
+      - string
+      - Uuid of page.
+    * -
+      - title
+      - string
+      - Title of page.
+    * -
+      - url
+      - string
+      - URL of page.
+    * -
+      - path
+      - int
+      - path of page
```

Comment:
Sometimes there is a period at the end, sometimes not. For these short texts I would prefer no period, because it's not a full sentence